internal/slicepool/slicepool.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2020 Jigsaw Operations LLC
+// Copyright 2020 Google LLC
```

Comment:
@fortuna @maddyhof  Shall we include both `Copyright 2020 Jigsaw Operations LLC, Google LLC` or just `Google LLC`?

Reply:
It should be either Outline Authors ([example](https://github.com/Jigsaw-Code/outline-client/blob/46b3a10d2158f2290b9e986140d054b5203354b5/src/www/app/main.ts#L1)) or Jigsaw Operations LLC. Not Google.

Jigsaw gives us more visibility. Though it may be worth it discussing with Legal.

Reply:
Let me know if you want me to ask Legal. I think consistency with what we have had in Outline code in the past is a good place to start.

Reply:
Great, I will revert these changes and ignore the corresponding `cross` errors.

Reply:
Updated, `cross` reported 37 copyright errors, and I will just ignore them. No other errors reported.